src/main/resources/resources/ApplicationResources_pt.properties

```diff
@@ -6945,4 +6945,7 @@ error.Registration.admission.phase.has.to.be.positive.number = A fase de entrada
 error.StudentCandidacy.admission.phase.has.to.be.positive.number = A fase de entrada de uma candidatura precisa de ser um n�mero positivo.
 
 label.dismissalLog = Credita��o
+
+error.dynamicFieldTag.duplicated.code = C�digo j� existe
+dynamicFieldTag.defaultName = Informa��es Adicionais
```

Comment:
It's better in the singular since is the term we're using now: _Informação Adicional_